sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/CosmosDiagnosticsLoggerConfig.java

```diff
@@ -0,0 +1,101 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+package com.azure.cosmos;
+
+import java.time.Duration;
+
+/**
+ * Configuration options that determine how the Cosmos diagnostics logger decides whether to log diagnostics
+ * for an operation or not
+ */
+public final class CosmosDiagnosticsLoggerConfig {
+    private Duration pointOperationLatencyThreshold;
+    private Duration feedOperationLatencyThreshold;
+    private float requestChargeThreshold;
+
+    /**
+     * Creates an instance of the CosmosDiagnosticsLoggerConfig class with default parameters
+     */
+    public CosmosDiagnosticsLoggerConfig() {
+        this(Duration.ofSeconds(1), Duration.ofSeconds(3), 1000);
+    }
+
+    /**
+     * Creates an instance of the CosmosDiagnosticsLoggerConfig class with custom parameters
+     * @param pointOperationLatencyThreshold the latency threshold for all point operations
+     * @param feedOperationLatencyThreshold the latency threshold for feed operations (query and change feed - and bulk)
```

Comment:
are readMany operations considered feed operations or point operations?

Reply:
Yes - readMany is effectively a query behind the scenes - will rename feedOperationLatencyThreshold to nonPointOperationLatencyThreshold i guess to avoid confusion